portugues/titanic/arrumar/middleware/404.md

```diff
@@ -0,0 +1 @@
+Infelizmente você recebe uma exceção de java.langOutOfMemory exception e acaba cortando os pulsos com os pedaços de uma xícara de café quebrada.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.